src/components/search-lightbox/script.js

```diff
@@ -1,19 +1,24 @@
 import api from '@/api';
+import ListPodcast from '@/components/list-podcast/list-podcast.vue';
+import SearchInput from '@/components/search-input/search-input.vue';
 
 export default {
   name: 'SearchLightbox',
+  components: {
+    ListPodcast,
+    SearchInput,
+  },
   data() {
     return {
       dialog: false,
       loading: false,
-      term: null,
       results: [],
     };
   },
   methods: {
-    async search() {
+    async searchTracks(term) {
       this.loading = true;
-      const results = await api.search(this.term);
+      const results = await api.search(term, true);
```

Comment:
Probably we shall think about a global debug flag to enable this.

Reply:
Outdated: fixed in 151d2a1